Apps/frontend/src/components/DistributeOffsetButton.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  import { NUM_CHANNELS } from "../const.js";
+  export let activeChannelCount;
+
+  const dispatch = createEventDispatcher();
+  const handleClick = async () => {
+    dispatch("distributeOffset", { offset: calculateOffset() });
+  };
+  function calculateOffset() {
```

Comment:
I'd much prefer to pass `activeChannelCount` as a function parameter instead of as an `export let`. The function is the only place we need the counter.

Reply:
Fixed: The button calculates the activeChannelCount itself by accessing the channelActivated store.